### DISCUSSION TITLE
Migration from Citadel node

### DISCUSSION THREAD

**Discussion by YuckFouBTC** [minibolt-guide/minibolt#82] [Q&A] [answered]
I have been running a Citadel node (https://github.com/runcitadel/citadel) for a while now.

I have many lightning channels already established.

I want to move away from Citadel and try minibolt.

Would it be possible to keep my existing channels? : https://pastebin.com/6PvUG9ku

**Comment by twofaktor** [comment, marked as answer]
Already on [roadmap](https://github.com/orgs/minibolt-guide/projects/1?pane=issue&itemId=41919567) only between MiniBolt->MiniBolt enviroment, it won't give support to migrate from the other platforms/environment because there are too many

**Reply by YuckFouBTC** [reply]
Wow, cool project tracking!  First time I've seen it on github.  Thanks!
